platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -3512,6 +3533,11 @@
             $('#importInstanceModal').modal('show');
         };
 
+        $scope.deleteSelectedInstances = function() {
+            $scope.checkedEntitiesJSON = $scope.getCheckedEntitiesJSON();
+            $("#deleteSelectedInstancesModal").modal('show');
```

Comment:
If we are changing this modal to BootsrapDialog3, we might even need to change all the other modals to that type because that looks better. As you said we will change this MDS module is moved to MOTECH-UI or  I could create sub-task and do it now.